### PR TITLE
[DDO-1467] Liquibase chart downstream changes

### PR DESCRIPTION
**Pull request by jack-r-warren**
Downstream effects of #503. Check out Argo CD to see this working in rawls-dev, leonardo-dev, and buffer-dev courtesy of https://github.com/broadinstitute/terra-helmfile/pull/1708.

### CODE REVIEW

File: charts/rawls/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: rawls
-version: 0.20.0
+version: 0.21.0
```
Comment (knapii-developments) [inline charts/rawls/Chart.yaml:3]: Why did you bump this version? Doesn't chart releaser do this for us?

File: charts/rawls/values.yaml [hunk 2]
```diff
@@ -101,24 +101,34 @@ deploymentDefaults:
...
+            name: "rawls-backend-sqlproxy-env"
+    migrationImage: "gcr.io/broad-dsp-gcr-public/rawls"
+    migrationArgsClasspath:
+      - "$(find /rawls -name 'rawls*.jar')"
+    migrationConfigFileMount:
+      secretName: "rawls-backend-app-ctmpls"
+    migrationDatabaseCredentials:
+      fromVaultSecret:
+        path: null
+        usernameKey: "slick_db_user"
```
Comment (knapii-developments) [inline charts/rawls/values.yaml:133]: This makes me happy. haha

File: charts/rawls/Chart.yaml (hunk 1 above)
Comment (jack-r-warren) [reply charts/rawls/Chart.yaml:3]: I bumped it here to give me a fresh version number, I was having trouble getting Argo to respect the versions I was giving it through helmfile. I know that the chart releaser will bump it again

File: charts/rawls/values.yaml (hunk 2 above)
Comment (jack-r-warren) [reply charts/rawls/values.yaml:133]: yeah slick is the name of the database thing they use in the code, suppose it made its way to their vault config

**Pull request closed without merging**